pkg/runctl/controller.go

```diff
@@ -494,8 +479,9 @@ func (c *Controller) handleAborted(pipelineRun k8s.PipelineRun) error {
 	intent := pipelineRun.GetSpec().Intent
 	if intent == api.IntentAbort && pipelineRun.GetStatus().Result == api.ResultUndefined {
 		pipelineRun.UpdateMessage("Aborted")
-		pipelineRun.UpdateResult(api.ResultAborted)
-		return c.changeState(pipelineRun, api.StateCleaning)
+		now := metav1.Now()
+		pipelineRun.UpdateResult(api.ResultAborted, now)
+		return c.changeState(pipelineRun, api.StateCleaning, now)
```

Comment:
```suggestion
		return c.setCleaningAndCountResult(pipelineRun, api.ResultAborted, now)
```
You could use this here